apps/condo/domains/banking/schema/BankContractorAccount.js

```diff
@@ -79,6 +80,17 @@ const BankContractorAccount = new GQLListSchema('BankContractorAccount', {
             type: Json,
         },
 
+        relatedTransactions: {
```

Comment:
Current name is not matched with description

```suggestion
        transactionsWithoutCostItems: {
```